Add history page (#994)
nasa-gcn/gcn.nasa.gov: app/routes/docs/history.mdx

---
handle:
  breadcrumb: History
---

import history from '~/img/gcn_history.png'

# History of GCN

<img
  src={history}
  width="900"

>   height="250"

Reviewer: This doesn't match the dimensions of the image file.

Author: updated with linearly scaled image

>   width="680"
>   height="230"

Reviewer: ```suggestion
  width="3304"
  height="1119"
  className="width-full height-auto"
```

  loading="lazy"
  alt="Timeline of GCN history"
/>

> The development of GCN began in 1992 as an innovative solution to a major hardware failure aboard the Compton Gamma-ray Observatory ([CGRO](https://heasarc.gsfc.nasa.gov/docs/cgro/cgro.html), 1991-2000). The mission was designed such that data was stored on a spacecraft tape recorder and downloaded several times per day via scheduled contacts with NASA's Tracking and Data Relay ([TDRS](https://www.nasa.gov/directorates/heo/scan/services/networks/tdrs_main/)) satellites, which then relayed the data to a ground station. Less than two years into the CGRO mission the tape recorder failed, threatening the continuation of this Great Observatory. To save CGRO, NASA arranged for continuous, real-time transmission of the science data through the TDRS system.

Reviewer: Which link for CGRO? https://heasarc.gsfc.nasa.gov/docs/cgro/cgro.html  or https://heasarc.gsfc.nasa.gov/docs/cgro/?

Author: The docs/cgro page has more comprehensive information about the instruments and data archives.  I'll update the link.

## BACODINE

> Dr. Scott Barthelmy at NASA/GSFC realized that this new data mode created an opportunity -- near-time notification of gamma-ray bursts detected by CGRO's Burst And Transient Search Experiment ([BATSE](https://heasarc.gsfc.nasa.gov/docs/cgro/batse/)) instrument. He went on to launch the BAtse COordinates DIstribution NEtwork (BACODINE) system in 1993. Software monitored the real-time telemetry of BATSE and produced an automatic localization within seconds, with a later improvement of a more accurate localization produced by BATSE scientists, after a delay of tens of minutes. The BATSE scientist were notified by BACODINE via pager when a transient occurred. They localized the source and sent the information to the BACODINE system for distribution.

Reviewer: Subject/verb agreement error:

```suggestion
Dr. Scott Barthelmy at NASA/GSFC realized that this new data mode created an opportunity -- near-time notification of gamma-ray bursts detected by CGRO's Burst And Transient Search Experiment ([BATSE](https://heasarc.gsfc.nasa.gov/docs/cgro/batse/)) instrument. He went on to launch the BAtse COordinates DIstribution NEtwork (BACODINE) system in 1993. Software monitored the real-time telemetry of BATSE and produced an automatic localization within seconds, with a later improvement of a more accurate localization produced by BATSE scientists, after a delay of tens of minutes. The BATSE scientists were notified by BACODINE via pager when a transient occurred. They localized the source and sent the information to the BACODINE system for distribution.
```

> The [GCN Notices](/notices) distribution mechanisms expanded over the following years: phone connections (July 1993), e-mail (December 1993), direct computer-to-computer socket connections over the Internet (January 1994), and by alpha-numeric pagers (September 1994) to any interested follow-up observatories within seconds of detection.

Reviewer: ```suggestion
The [GCN Notices](/notices) gained new distribution mechanisms over the following years: phone connections (July 1993), e-mail (December 1993), direct computer-to-computer socket connections over the Internet (January 1994), and by alpha-numeric pagers (September 1994) to any interested follow-up observatories within seconds of detection.
```

The first robotic follow-up instrument to connect to BACODINE was the Gamma-Ray Optical Counterpart Search Experiments ([GROCSE](https://ui.adsabs.harvard.edu/abs/1995Ap%26SS.231..255A/abstract)) in August 1993. Additional capabilities were added to BACODINE throughout the mid 1990's including trigger classification (GRBs from non-GRBs), monitoring (including 24/7 staffing) and the modern incarnation of the interplanetary network ([IPN](https://heasarc.gsfc.nasa.gov/docs/heasarc/missions/ipn.html)).

## Gamma-ray Burst Coordinates Network

> New instruments were added to the network, including the Rossi X-ray timing Explorer ([RXTE](https://heasarc.gsfc.nasa.gov/docs/xte/XTE.html)) Proportional Counter Array ([PCA](https://heasarc.gsfc.nasa.gov/docs/xte/PCA.html)) and the Array of Low Energy X-Ray Imaging Sensors ([ALEXIS](https://heasarc.gsfc.nasa.gov/docs/heasarc/missions/alexis.html)) extreme ultraviolet transient mission. GCN Notices from the CGRO Imaging Compton Telescope ([COMPTEL](https://heasarc.gsfc.nasa.gov/docs/cgro/comptel/)) were added in November 1997. In recognition of the broader usage, the system was renamed the Gamma-ray burst Coordinates Network (GCN) in 1997.

Reviewer: The sentence begins, "New instruments were added to the network," but of the two examples given, one is an instrument and one is a mission.

> New instruments were added to the network, including the Rossi X-ray timing Explorer ([RXTE](https://heasarc.gsfc.nasa.gov/docs/xte/learning_center/index.html)) Proportional Counter Array ([PCA](https://heasarc.gsfc.nasa.gov/docs/xte/learning_center/pca.html)) and the Array of Low Energy X-Ray Imaging Sensors ([ALEXIS](https://heasarc.gsfc.nasa.gov/docs/heasarc/missions/alexis.html)) extreme ultraviolet transient mission. GCN Notices from the CGRO Imaging Compton Telescope ([COMPTEL](https://heasarc.gsfc.nasa.gov/docs/cgro/comptel/)) were added in November 1997. In recognition of the broader usage, the system was renamed the Gamma-ray burst Coordinates Network (GCN) in 1997.

Reviewer: Review these links. Some of these look like public outreach pages. For example, how about this for XTE? https://heasarc.gsfc.nasa.gov/docs/xte/XTE.html

An early, important discovery came from the transmission of the [BeppoSAX localization](https://gcn.nasa.gov/circulars/199) of GRB 990123 by the GCN system. This GRB location automatically steered the robotic telescope [ROTSE-I](https://gcn.nasa.gov/circulars/205) to GRB 990123 only 22 seconds after the start of the GRB, leading to the first detection of an optical flash during the prompt phase of a GRB.

[GCN Circulars](/circulars) -- the curated observation reports were introduced in 1997, and has led to >35k circulars as of mid-2023. These citable, though non-peer reviewed, reports appear automatically in ADS and are regularly cited in the astronomical literature.

## Gamma-ray Coordinates Network

As GCN grew in its utilization beyond the GRB community for other types of high-energy astronomical transients (e.g. soft gamma-ray repeaters, tidal disruption events), it was renamed once again to the Gamma-ray Coordinates Network (also GCN) in ~2009.

> GCN continued to play a seminar role in enabling astrophysical discovery with the distribution of GCN Notices from the advanced gravitational wave network and astrophysical neutrinos from IceCube starting in ~2015. This included a completely parallel GCN Notices and Circulars system while gravitational alerts were distributed only to teams with MOUs with the LIGO Virgo Consortium, including the [first detection of a binary neutron star merger](https://gcn.nasa.gov/circulars?page=2&query=G298048) and its subsequent multiwavelength counterparts.

Reviewer: ```suggestion
GCN continued to play a seminar role in enabling astrophysical discovery with the distribution of GCN Notices from the advanced gravitational wave network and astrophysical neutrinos from IceCube starting in ~2015. This included a completely parallel GCN Notices and Circulars system while gravitational alerts were distributed only to teams with MOUs with LIGO and Virgo, including the [first detection of a binary neutron star merger](https://gcn.nasa.gov/circulars?page=2&query=G298048) and its subsequent multiwavelength counterparts.
```

Reviewer: Also, we don't define the acronym MOU. Consider instead "agreements" to avoid the acronym.

> The principle visionary and tireless developer of GCN over its first 30 years was Scott Barthelmy (NASA/GSFC) with significant contributions from Teresa Sheets, Craig Markwardt and Tom McGlynn, with [many others](https://gcn.gsfc.nasa.gov/people.html) making significant contributions to design and operation.

Reviewer: "Visionary" is an accurate word for Scott's contribution, but it may come across as self-aggrandizing given that this copy is on our own web site. Is there another way that we can say this?

Author: We could change "principle visionary" to "principal investigator".  It's more of a technical term, but makes it clear that he pushed this effort along for all of those years.

> ## The General Coordinates Network

Reviewer: We have some really good copy in the presentation about the scientific and technological motivation for the General Coordinates Network. Would you adapt some of that material here?

> In the tradition of renaming the GCN, but maintaining the acronym, the General Coordinates Network (also GCN; [https://gcn.nasa.gov](https://gcn.nasa.gov)) debuted in [July 2022](/news) serving GCN notices via a cloud-operated Kafka broker. The legacy GCN system, now known as GCN Classic, is being maintained in parallel to the new GCN, but features are gradually being migrated. GCN Notices became available via email in September 2022 with self-service subscriptions. The GCN Circulars service was migrated in its entirety from GCN Classic to the new GCN in [April 2023](/news).

Reviewer: ```suggestion
In the tradition of renaming the GCN, but maintaining the acronym, the General Coordinates Network (also GCN; [https://gcn.nasa.gov](https://gcn.nasa.gov)) debuted in [July 2022](/news) distributing GCN Notices from a cluster of Kafka brokers in the cloud. The legacy GCN system, now known as GCN Classic, is being maintained in parallel to the new GCN, but features are gradually being migrated. GCN Notices became available via email in September 2022 with self-service subscriptions. The GCN Circulars service was migrated in its entirety from GCN Classic to the new GCN in [April 2023](/news).
```

> In the tradition of renaming the GCN, but maintaining the acronym, the General Coordinates Network (also GCN; [https://gcn.nasa.gov](https://gcn.nasa.gov)) was debuted in [July 2022](/news) serving GCN notices via a cloud-operated Kafka broker. The legacy GCN system, now known as GCN Classic, is being maintained in parallel to the new GCN, but features are gradually being migrated. GCN Notices became available via email in September 2022 with self-service subscriptions. The GCN Circulars service was migrated in its entirety from GCN Classic to the new GCN in [April 2023](/news).

Reviewer: ```suggestion
In the tradition of renaming the GCN, but maintaining the acronym, the General Coordinates Network (also GCN; [https://gcn.nasa.gov](https://gcn.nasa.gov)) debuted in [July 2022](/news) serving GCN notices via a cloud-operated Kafka broker. The legacy GCN system, now known as GCN Classic, is being maintained in parallel to the new GCN, but features are gradually being migrated. GCN Notices became available via email in September 2022 with self-service subscriptions. The GCN Circulars service was migrated in its entirety from GCN Classic to the new GCN in [April 2023](/news).
```

> Over the next few years, GCN Classic services will be fully migrated to the new GCN. New Notice types will only be available via the new GCN, and we look forward to many [new capabilities](/about/roadmap) and the growth of our GCN community.

Reviewer: ```suggestion
Over the next few years, GCN Classic services will be fully migrated to the new GCN. New Notice types will only be available via the new GCN, and we look forward to many [new capabilities](/docs/roadmap) and the growth of our GCN community.
```
